TODO/force-with-lease.md

```diff
@@ -3,44 +3,44 @@
 > * 原文作者：[Steve Smith](https://legacy-developer.atlassian.com/blog/authors/ssmith/)
 > * 译文出自：[掘金翻译计划](https://github.com/xitu/gold-miner)
 > * 本文永久链接：[https://github.com/xitu/gold-miner/blob/master/TODO/force-with-lease.md](https://github.com/xitu/gold-miner/blob/master/TODO/force-with-lease.md)
-> * 译者：
+> * 译者：[LeviDing](https://github.com/leviding)
 > * 校对者：
 
-# -force considered harmful; understanding git's --force-with-lease
+# 使用 `-force` 被认为是有害的；了解 Git 的 `-force-with-lease` 命令
 
-Git's `push --force` is destructive because it unconditionally overwrites the remote repository with whatever you have locally, possibly overwriting any changes that a team member has pushed in the meantime. However there is a better way; the option --force-with-lease can help when you do need to do a forced push but still ensure you don't overwrite other's work.
+Git 的 `push --force` 是破坏性的，因为它无条件地覆盖远程存储库，无论你在本地拥有什么。使用这个命令，可能覆盖团队成员在此期间推出的所有更改。但是这有一个更好的方法，当你需要强制推送，但仍需确保不覆盖其他人的工作时，`-force-with-lease` 可以帮助到你。
 
-![I don't always push --force...](https://developer.atlassian.com/blog/2015/04/force-with-lease/force-with-lease.jpg)
+![我不经常强制推送...](https://developer.atlassian.com/blog/2015/04/force-with-lease/force-with-lease.jpg)
 
-It's well known that git's `push --force` is strongly discouraged as it can destroy other commits already pushed to a shared repository. This isn't always completely fatal (if the changes are in someone's working tree then they can be merged), but at the very least it's inconsiderate, at worst disastrous. This is because the `--force` option makes the head of the branch point at your personal history, ignoring any changes that may have occurred in parallel with yours.
+众所周知，git 的 `push -force` 是非常不鼓励使用的，因为它可以破坏已经推送到公共仓库的其他的提交。虽然这不总是完全致命的（如果变化是在某人的工作树中，那么他们可以再来合并），但至少使用这个命令是非常轻率的，最糟糕的时候可能是灾难性的。这是因为 `--force` 选项使分支的头部位于你个人的历史记录上，忽略可能与你的并行发生的任何更改。
```

Comment:
> This is because the `--force` option makes the head of the branch point at your personal history, ignoring any changes that may have occurred in parallel with yours.

这是因为 `--force` 指令选项迫使分支的头指针指向你个人的修改记录，而忽略了那些其他和你同时进行地更改。